src/app/configuration/configuration.component.ts

```diff
@@ -66,7 +66,8 @@ export class ConfigurationComponent implements OnInit {
       localDomain: this.defaultLocal,
       name: [
         '',
-        Validators.required,
+        [ Validators.required,
```

Comment:
planet name does not have any character restriction right now except it should not start with special characters